src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -5,4 +5,6 @@
 
 public interface MsgService {
     String generateMsg(String msgType, JsonObject bodyJson);
+    String validateMsg(String msgType, JsonObject bodyJson);
```

Comment:
These methods are now removed as they have been replaced by new method in version 0.1.4